modules/core/shared/src/main/scala/gem/math/MagnitudeValue.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.math
+
+import cats.{ Order, Show }
+import cats.instances.int._
+
+/**
+ * Exact magnitude value represented as an int with the original value scaled up
+ * @param value This magnitude integral value
+ */
+final case class MagnitudeValue(private[gem] val scaledValue: Int) extends Product with Serializable {
+  def toDouble: Double = scaledValue / 100.0
+}
+
+object MagnitudeValue {
+
+  final lazy val ZeroMagnitude = MagnitudeValue(0)
+
+  /** @group Typeclass Instances */
+  implicit val MagnitudeValueShow: Show[MagnitudeValue] =
+    Show.fromToString
```

Comment:
I would prefer that the `toString` show the value humans want to see; something like `s"MagnitudeValue($toDouble)"`